extensions/wikia/MercuryApi/handlers/MercuryApiArticleHandler.class.php

```diff
@@ -95,6 +95,31 @@ public static function getArticleJson( WikiaRequest $request, Article $article )
 		)->getData();
 	}
 
+	public static function getFeaturedVideoDetails( Title $article ): array {
+		global $wgEnableArticleFeaturedVideo, $wgArticleVideoFeaturedVideos;
+
+		$title = $article->getPrefixedDBkey();
```

Comment:
This one definitely is not `title` ;)
Please rename those variables to make it less confusing.